epoxy-processor/src/main/java/com/airbnb/epoxy/AttributeInfo.java

```diff
@@ -142,6 +155,43 @@ private boolean isFieldPackagePrivate(Element attribute) {
   }
 
   /**
+   * Checks if the given private field has getter and setter for access to it
+   */
+  private void findGetterAndSetterForPrivateField(ErrorLogger errorLogger) {
+    for (Element element : classElement.getEnclosedElements()) {
+      if (element.getKind() == ElementKind.METHOD) {
+        ExecutableElement method = (ExecutableElement) element;
+        String methodName = method.getSimpleName().toString();
+        // check if it is a valid getter
+        if (methodName.equalsIgnoreCase(String.format("get%s", name))
+            || methodName.equalsIgnoreCase(String.format("is%s", name))
```

Comment:
wow, thanks! That makes sense now. Yeah, I've seen this article, but I was pretty sure that the code is alright and the issue was in errors handling. Shame on me :P